src/com/hms_networks/americas/sc/historicaldata/HistoricalDataQueueManager.java

```diff
@@ -115,7 +131,7 @@ public static boolean doesTimeTrackerExist() {
    * @return time tracker file value
    */
   public static long getCurrentTimeTrackerValue() throws IOException {
-    String startTimeTrackerMs = FileAccessManager.readFileToString(timeMarkerFileName);
+    String startTimeTrackerMs = FileAccessManager.readFileToString(timeMarkerFile1Name);
```

Comment:
Same comment: Shouldn't this method be updated to handle the new multiple file logic?